code/datums/factions/upp.dm

```diff
@@ -50,11 +50,15 @@
 		list("Type 64 Submachinegun", 20, /obj/item/weapon/gun/smg/bizon/upp, null, VENDOR_ITEM_REGULAR),
 		list("Type 71 Pulse Rifle", 30, /obj/item/weapon/gun/rifle/type71, null, VENDOR_ITEM_REGULAR),
 		list("Type 71 Pulse Rifle Carbine", 30, /obj/item/weapon/gun/rifle/type71/carbine, null, VENDOR_ITEM_REGULAR),
+		list("Type 23 Riot Shotgun", 25, /obj/item/weapon/gun/shotgun/type23, null, VENDOR_ITEM_REGULAR),
```

Comment:
from what i remember Riot Shotgun is supposed to be a specialist weapon for UPP not a common